z_infra_provisioning/cloud_infra_center/ocp_upi/destroy-volumes.yaml

```diff
@@ -0,0 +1,84 @@
+# =================================================================
+# Copyright 2021 https://github.com/openshift/installer
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =================================================================
+
+# =================================================================
+# Licensed Materials - Property of IBM
+#
+# (c) Copyright IBM Corp. 2021 All Rights Reserved
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# =================================================================
+#
+# Change Summary: 
+# - Delete unnecessary Ansible tasks
+# - Read compute nodes names from json
+
+# Required Python packages:
+#
+# ansible
+# openstackclient
+# openstacksdk
+
+- hosts: localhost
+  gather_facts: no
+
+  tasks:
+  - name: 'Import common yaml'
+    import_tasks: common.yaml
+
+  - name: 'Delete bootstrap boot volumes in KVM'
+    os_volume:
+      state: absent
+      name: "{{ os_bootstrap_server_name }}-boot"
+    when: 
+    - vm_type == "kvm"
+    - volume_type_id is defined
+
+  - name: 'Get controller volumes from ICIC command line'
+    shell: |
+       openstack volume list -c Name -f value | grep {{ infra_id.stdout_lines[0] }} | grep master | cut -d '-' -f 1-3
```

Comment:
`openstack --os-volume-api-version=3 volume list` is required to avoid the volume-api-version=3.6 issue in ICIC.

Reply:
same in other places.